CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## Changed
+
+- Improved `go.sum` file parsing to ensure only actively used modules are 
+  reported and prevent the parser from listing unused packages
```

Comment:
```suggestion
- Improved `go.sum` file parsing to prevent the parser from listing unused packages
```

This is kinda saying the same thing twice. It's good to be concise and I think the latter part of your sentence accurately conveys the purpose of this change.